Add guidelines (#1)
PyLadiesGermany/PySV_PyLadies_fund: guideline.md

# Pyladies Germany PySV Grant Program

The annual €5000 grant program from the PySV for PyLadies aims to support the activities and growth of PyLadies chapters in Germany. It is essential to ensure that these funds are used transparently, therefore we have devised the following guidelines. The grant will be issued to PyLadies each year after the PyCon DE conference (circa June), once assigned the funds are available for disposal by the PyLadies using the following guidelines. Money not used within the year will stay on the PyLadies account and can continue to be used.

**Allowable expenses:**

> - Community events: this includes room rentals, refreshments, and similar costs.

Reviewer: should we add devices to be used on meetups events: microphone, camera, lightning?

> - Community events: this includes room rentals, refreshments, and similar costs.

Reviewer 2: ```suggestion
- Community events: this includes room rentals, refreshments, devices used for events (for example microphone, camera, lightning), and similar costs.
```

- Promotional materials: such as stickers, banners, T-shirts, and other marketing materials.

> - Software tools: Subscription-based tools or one-time software purchases that benefit the chapter (these may also be shared across chapters).

Reviewer: maybe we can use what PyLadies Berlin is using to share passwords and so on :)

Reviewer 2: yeh good shout, we could also have within this repo a list of those tools (and current state of them) then link that here? For Passwords we are using Lastpass and tbh I only "kept" the subscription because I forgot to cancel it and was then charged for it :(

Author: I made a separate readme to track shared tools and linked it here

- To pay/reimburse costs to speakers for PyLadies events
- Travel reimbursement for members to go to other events (note the regional restrictions)

**Disallowed Expenditures:**
- Regional Restrictions: Funds can only be spent on activities in the DE, CH, and AT regions as defined in the PySV Bylaws.
- Personal expenses or any activity not related to PyLadies.
- It should be avoided that the opportunities funded benefit repeatedly only a single individual or small group of the chapter members within a single year

## Accessing funds

Chapters wishing to apply for funds should:
- Open a ticket on our platform (to be set up) with a brief statement of the intended use of the funds, detailing the purpose and amount requested.

> - If necessary wait for approval (see process below)

Reviewer: should we estipulate approving time? to better manage expectations

Reviewer 2: agree, I am just not sure what is realistic for this. I have set up a Slack integration so we would get messaged about new requests and my gut feeling is fair is 2-3 business (week) days but I am not sure. What do you think @laysauchoa

We expect chapters with multiple groups within it to have transparency with one another and to help us achieve an equitable way of distributing funds.

### Request for larger amounts

Some chapters might prefer to request approval for a larger amount in one go and manage the details of how it is spent internally (specifics still need to be reported and individual invoices sent). This could be for example be requesting a proportion of the total grant for the year upfront or for a single initiative which will include multiple costs. Ideally, such initiatives may be special programs that are indicated as planned by the chapter or chapters and presented, whenever possible, at the beginning of the year in order to plan the annual budget. After approval, the chapter will be able to get reimbursements for up to this amount without the need for further approval.

> These chapters should already be established, with a track record of events they have run and (some) consistency in the organizing team (i.e. it should not be a whole new group who have recently taken over a chapter). It would be expected that this group would not receive approval for applications for further funds unless we have leftover budget (towards year end).

Reviewer 3: Should we say that for this kind of request a group needs to have run at least 5 events in the past 12 months? It's an arbitrary number but I'm trying to think from the perspective of a new chapter to give them more clarity on when they would be able to make these kinds of requests.

Reviewer 2: That's a great point, we want transparency and explicitness on what we mean!

The amount considered would be the total grant amount divided by the current number of active chapters + an additional slice for Germany as a whole. (i.e., if there are 4 active chapters and the grant is 5000 EUR a single chapter could request 1000EUR)
Process (differs slightly from above):
- Ticket created on the reporting platform for the total amount, including ideas of what it will cover (exact items should be listed afterward)
- Approval for the total amount by at least 2 representatives outside of the requesting chapter
- Individual invoices should be sent to the PySV with the PyLadies Germany email copied in

> - Reimbursement will happen in (X-X days)

Reviewer 3: Should we say within 14 days of approval?

Reviewer 2: I will check this with the PySV as they would control this not us.

Author: confirming 14 days

PyLadies Germany Spokesperson attendance of PySV meetings:

The spokesperson's involvement will also include attending PySV meetings. The frequency and schedule of these meetings will be determined jointly by PyLadies and PySV.

## Reporting:

The platform for approving and tracking requests should be as publicly open as possible so that any PyLady in Germany has access. Therefore no individual's personal information should be shared unless it is relevant (individuals can be referred to by their role i.e. event speaker). Invoices should not be posted there.

Tickets will be labeled by chapter and year and the platform should allow PyLadies to see at a glance what has been requested, approved, and reimbursed. Anyone should be able to make a request and there should be guidelines on how to do so.

A general update on notable events and activities should be sent to PySV every six months. This helps PySV understand the needs and accomplishments of PyLadies. The report should be sent by the spokesperson with the support of the other approval group members.

> Details of all expenditures must be reported so that accounts can be properly reconciled. These are sent directly to the PySV with the germany@pyladies.com email in CC.

Reviewer 2: ```suggestion
Details of all expenditures must be reported so that accounts can be properly reconciled. These are sent directly to the PySV with the [pyladies@python-verband.org](mailto:pyladies@python-verband.org) email in CC.
```

## Current Approvals Group

> Email: germany@pyladies.com (this is a Google group)

Reviewer 2: ```suggestion
Email: pyladies@python-verband.org
```

# Pyladies Germany PySV Grant Program

The annual €5000 grant program from the PySV for PyLadies aims to support the activities and growth of PyLadies chapters in Germany. It is essential to ensure that these funds are used transparently, therefore we have devised the following guidelines. The grant will be issued to PyLadies each year after the PyCon DE conference (circa June), once assigned the funds are available for disposal by the PyLadies using the following guidelines. Money not used within the year will stay on the PyLadies account and can continue to be used.

**Allowable expenses:**

> - Community events: this includes room rentals, refreshments, devices used for events (for example microphone, camera, lightning), and similar costs.

Reviewer: can we also add designer work?

- Promotional materials: such as stickers, banners, T-shirts, and other marketing materials.

> - Software tools: Subscription-based tools or one-time software purchases that benefit the chapter (these may also be shared across chapters - see [shared tools list](./Docs/shared_tools.md)).

Reviewer: can we expand shared tools? or what would be the process?

Reviewer 2: lets put the process for this in the shared tools doc as I guess it's somewhat separate to the grant request to add a new shared tool (and should happen before)

- To pay/reimburse costs to speakers for PyLadies events
- Travel reimbursement for members to go to other events (note the regional restrictions)
- Educational material for events or raffle prizes such as books, ebooks or vouchers for online courses 

**Disallowed Expenditures:**
- Regional Restrictions: Funds can only be spent on activities in the DE, CH, and AT regions as defined in the [PySV Bylaws](https://python-verband.org/documents/).
- Personal expenses or any activity not related to PyLadies.
- It should be avoided that the opportunities funded benefit repeatedly only a single individual or small group of the chapter members within a single year

## Accessing funds

Chapters wishing to apply for funds should:
- Open an issue in this repo, choosing the relevant ISSUE type and follow the outlined issue template, this will include a brief statement of the intended use of the funds, detailing the purpose and amount requested. You will also be prompted to add labels relevant to your request
- If necessary wait for approval (see process below)
- Submit an invoice directly to the PySV (invoice@python-verband.org) with the pyladies@python-verband.org in CC.

>   - Ensure that all expenditures are accompanied by a proper receipt. 

Reviewer 2: maybe clarify what is a proper receipt

## Approval Process

We want the approval process to be as quick as possible to reduce the load for individual organizers, at the same time we want to ensure that the funds are distributed in a way that maximizes our mission across Germany.

Approvals will be given by the approval group following the approval guidelines (see below). Requests will fall into one of the following approval categories:

### No approval required

For some things we want to ensure chapters can get the quickest access possible, this means the requirement for pre-approval is removed for eligible requests. The chapter must still create a ticket on our platform with relevant information but can immediately send an invoice to the PySV (with the PyLadies email copied in) These items include:
- Up to 200 EUR for stickers per chapter per year
- A roll-up banner (one-time purchase) for a chapter
- Shared subscriptions i.e. streaming software or password managers (these subscriptions should have been agreed upon by the chapters beforehand, you can find a list of shared subscriptions [here](Docs/shared_tools.md))

> These quick approvals apply to all existing and future chapters who have had at least one official event under the PyLadies brand or are attending PyCon DE as a group. Stickers for chapters might be ordered together (to reduce costs) and this should be organised between the chapters.

Reviewer 2: I think we should drop the future here, so there is no concern that folks not known to the community spend money from the fund without some discussion happening before hand

The amount considered would be the total grant amount divided by the current number of active chapters + an additional slice for Germany as a whole. (i.e., if there are 4 active chapters and the grant is 5000 EUR a single chapter could request 1000EUR)

**Process (differs slightly from above):**
- Open a [Approval Multi Request](https://github.com/PyLadiesGermany/PySV_PyLadies_fund/issues/new?assignees=sleepypioneer%2Claysauchoa%2Cdiscombobulateme%2Cpga99%2Cterezaif&labels=approval_request%2Capproval_multi_request&projects=&template=approval-multi-request.md&title=Approval+Multi+Request) issue in this repository for the total amount, including ideas of what it will cover (exact items should be listed afterward)
- Wait for approval for the total amount by at least 2 representatives outside of the requesting chapter

> - Individual invoices should be sent to the PySV with the PyLadies Germany email copied in

Reviewer 2: link email again here

- Reimbursement will happen within 14 days
- Individual spends should be added to the ticket on the reporting platform marked as reimbursed with an update to the amount left
- Add any additional information we can use for reporting after the funds are spent (optional but helps with reporting)

> - When the total amount has been reimbursed move the ticket to the reimbursed column. (consideration for further funds can then happen)

Reviewer: is there a ticket in some board or do you mean to take some action on the issue?

You can also check the process of your request at any time on our [project board](https://github.com/orgs/PyLadiesGermany/projects/2).

If a member of the approval group is repeatedly unable to fulfill this, they should look for a new representative from their chapter to fill the role. We aim to have at least 3 members of this group in addition to the spokesperson.

The Approval Group has the task not only of approving or denying requests but also of keeping track of the balance of the fund and encouraging its full use each year. We know that some chapters are more active or are simply located in larger cities and have more members, so they may need to apply for a larger portion of the fund. However, it is important to keep in mind that this funding can be the incentive needed to support smaller groups to be more active or to provide the tools and means needed for their members to be more active. Similarly, this concept should prevail when it comes to using funds to encourage support for marginalized groups within PyLadies, so that it reaches those that need more support or structure.

> 

Reviewer 2: what is the upper limit that a chapter can request?

Tickets will be labeled by chapter and year and the platform should allow PyLadies to see at a glance what has been requested, approved, and reimbursed. Anyone should be able to make a request and there should be guidelines on how to do so.

A general update on notable events and activities should be sent to PySV every six months. This helps PySV understand the needs and accomplishments of PyLadies. The report should be sent by the spokesperson with the support of the other approval group members.

Details of all expenditures must be reported so that accounts can be properly reconciled. These are sent directly to the PySV with the [pyladies@python-verband.org](mailto:pyladies@python-verband.org) email in CC.

## Current Approvals Group

Email: germany@pyladies.com (this is a Google group)

*Access to the email is only granted to current approval group members and should have 2FA enabled.*

Approval Platform: GitHub Project (this is public and should contain no personal information)

**Spokesperson:** [Jessica Greene](https://github.com/sleepypioneer/) [Berlin]
XXXX XXXX [Berlin]
XXXX XXXX [Hamburg]

> XXXX XXXX [Munich]

Reviewer: ```suggestion
[Laysa Uchoa](https://github.com/laysauchoa/) [Munich]
```
should we add our names here?

Tickets will be labeled by chapter and year and the platform should allow PyLadies to see at a glance what has been requested, approved, and reimbursed. Anyone should be able to make a request and there should be guidelines on how to do so.

A general update on notable events and activities should be sent to PySV every six months. This helps PySV understand the needs and accomplishments of PyLadies. The report should be sent by the spokesperson with the support of the other approval group members.

Details of all expenditures must be reported so that accounts can be properly reconciled. These are sent directly to the PySV with the [pyladies@python-verband.org](mailto:pyladies@python-verband.org) email in CC by the fund spokesperson in regular intervals (TBD: which interval, likely 6 months or yearly).

## Current Approvals Group

Email: germany@pyladies.com (this is a Google group)

*Access to the email is only granted to current approval group members and should have 2FA enabled.*

Approval Platform: GitHub Project (this is public and should contain no personal information)

**Spokesperson:** [Jessica Greene](https://github.com/sleepypioneer/) [Berlin]

> XXXX XXXX [Berlin]

Reviewer 2: @discombobulateme are you happy to be added here

> XXXX XXXX [Berlin]

Reviewer 2: ```suggestion
[Paloma Oliveira](https://github.com/discombobulateme) [Berlin]
```

> XXXX XXXX [Hamburg]

Reviewer 2: @terezaif are you happy to be added here for now? We can also leave Hamburg blank for the time being

> XXXX XXXX [Hamburg]

Reviewer 2: ```suggestion
[Tereza Iofciu](https://github.com/terezaif) [Hamburg]
```

[Laysa Uchoa](https://github.com/laysauchoa/) [Munich]

> XXXX XXXX [SudWest]

Reviewer 2: @pga99 are you happy to be added here?

> XXXX XXXX [SudWest]

Reviewer 2: ```suggestion
[Paula González Avalos](https://github.com/pga99) [SudWest]
```
